acct-group/iipsrv/iipsrv-0-r1.ebuild

```
@@ -0,0 +1,8 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit acct-group
+
+ACCT_GROUP_ID=536
```

```suggestion
ACCT_GROUP_ID=-1
```
When you are adding it to guru, please set the group id to -1